gatsby-config.js

```diff
@@ -312,8 +312,8 @@ const gatsbyConfig = {
     {
       resolve: 'gatsby-source-contentful',
       options: {
-        spaceId: process.env.CONTENTFUL_SPACE,
-        accessToken: process.env.CONTENTFUL_TOKEN,
+        spaceId: process.env.CRDT_CONTENTFUL_SPACE,
```

Comment:
@duynguyen158 I think the builds aren't working because of these new env variables -- neither Circle CI nor Netlify can see a `CRDT_CONTENTFUL_TOKEN` / `_SPACE` build variable, so they both fail on builds. (Also, they can't see the expected `CONTENTFUL_TOKEN` / `_SPACE` tokens since those lines are removed.)

Maybe we could add that variable to get this launched @kevee?

Also, Circle CI is also failing on the lint task:
```
/tmp/workspace/src/components/pages/race/reports/content-blocks/report-viz.js
  47:5  error  Arrow function expected no return value  consistent-return
```